add integration to Amplify (#68)
logzio/logzio_aws_serverless: python3/amplify/README.md

| TOKEN (Required)                               | The [token](https://app.logz.io/#/dashboard/settings/general) of the account you want to ship to.                                                                                                                                                                                                                                                           |
| LISTENER_URL (Required)                        | Determines protocol, listener host, and port. For example, `https://<<LISTENER-HOST>>:8071`. <br > Replace `<<LISTENER-HOST>>` with your region's listener host (for example, `https://listener.logz.io:8071`). For more information on finding your account's region, see [Account region](https://docs.logz.io/user-guide/accounts/account-region.html) . |

> | AMPLIFY_DOMAIN (Required)                      | Amplify domain URL can find in amplify adming dashboard in section **General** inside **Production branch URL**                                                                                                                                                                                                                                             |

Reviewer: ```suggestion
| AMPLIFY_DOMAIN (Required)                      | Amplify domain URL can be found in the Amplify admin dashboard in **General** under **Production branch URL**.                                                                                                                                                                                                                                             |
```

| TYPE (Default: `logzio_amplify_access_lambda`) | The log type you'll use with this Lambda. This can be a [type that supports default parsing](https://docs.logz.io/user-guide/log-shipping/built-in-log-types.html), or a custom log type. <br> You'll need to create a new Lambda for each log type you use.                                                                                                |

> | AMPLIFY_APP_ID (Required)                      | You can find app id inside your amplify admin dashboard in section general isnide you App ARN field arn:aws:amplify:`REGION`:`AWS_ID`:apps/`APP_ID`                                                                                                                                                                                                         |

Reviewer: ```suggestion
| AMPLIFY_APP_ID (Required)                      | You can find the app ID in your Amplify admin dashboard in **General** under the **App ARN** field arn:aws:amplify:`REGION`:`AWS_ID`:apps/`APP_ID`.                                                                                                                                                                                                         |
```

| TIMEOUT                                        | For trigger lambda function we define (in minutes) timeout to fetch Amplify logs for range time of TIMEOUT                                                                                                                                                                                                                                                  |

#### 4. Set the EventBridge (CloudWatch Events) trigger

1. Find the **Add triggers** list (left side of the Designer panel) and choose **EventBridge (CloudWatch Events)** from this list.

> 2. If you don't have pre defined schedule type(ex 1min) In the **Rule** click on create new rule.

Reviewer: ```suggestion
2. If you don't have pre-defined schedule type (e.g. 1min) In the **Rule**, click **Create new rule**.
```

> 3. **Rule name** enter a name to uniquely identify your rule.

Reviewer: ```suggestion
3. In **Rule name**, enter a name to uniquely identify your rule.
```

> 4. **Rule description** (Optional) Provide an optional description for your rule.

Reviewer: ```suggestion
4. In **Rule description**, if required, provide an optional description for your rule.
```

> 5. **Rule type** choose Schedule expression that suppose to be equal to the TIMEOUT of enviroment variable (ex. rate(5 minutes))

Reviewer: ```suggestion
5. In **Rule type**, choose the schedule expression that is equal to the TIMEOUT of the environment variable (e.g., 5 minutes).
```

> 5. **Rule type** choose Schedule expression that suppose to be equal to the TIMEOUT of enviroment variable (ex. rate(5 minutes))

Reviewer: ```suggestion
5. In **Rule type**, choose the schedule expression that is to the TIMEOUT of environment variable (e.g., 5 minutes).
```

#### 5. Update Permissions for Lambda Function

> 1. Go to **Configuration** in your Lamnda function find there tab **Permissions**

Reviewer: ```suggestion
1. Go to **Configuration** in your Lambda function and select the **Permissions** tab.
```

> 2. Click on A role name see in example **lambda-basic** it will redirect you to the IAM> Roles> lambda-basic

Reviewer: ```suggestion
2. Click the role name shown in the example **lambda-basic**. It will redirect you to the **IAM> Roles> lambda-basic**.
```

   ![Lambda-role](img/lambda-role.png)

> 3. On a role page inside Permissions tabs you will see dropdown **Add permissions** and click on **Create inline policy** it will redirect you to Create Policy page

Reviewer: ```suggestion
3. On the role page inside the **Permissions** tab, select the dropdown **Add permissions** and click on **Create inline policy**. It will redirect you to the **Create Policy** page.
```

   ![Create-inline](img/create-inline.png)

> 4. On Create Policy page you will see 2 tabs and you need to choose JSON tab

Reviewer: ```suggestion
4. On the **Create Policy** page, select the **JSON** tab.
```

   ![JSON-tab](img/json-tab.png)

> 5. On A JSON tab please fill in JSON with your parameters

Reviewer: ```suggestion
5. Fill in JSON with your parameters as follows:
```

```
{
    "Version": "2012-10-17",
    "Statement": [
		{
            "Effect": "Allow",
            "Action": [
                "amplify:GenerateAccessLogs"
            ],
            "Resource": "arn:aws:amplify:AWS_REGION:XXX66029XXXX:apps/XXXXdn0mprXXXX/accesslogs/*"
        }
	]
}
```

> AWS_REGION where your Amplify App is located(ex. us-west-2)

Reviewer: ```suggestion
Replace `AWS_REGION` with the AWS region of your Amplify App (e.g.,us-west-2)
```

> XXX66029XXXX with your AWS Account Id

Reviewer: ```suggestion
Replace `XXX66029XXXX` with your AWS Account ID.
```

> XXXXdn0mprXXXX with the AWS Amplify App Id

Reviewer: ```suggestion
* Replace `XXXXdn0mprXXXX` with the AWS Amplify App ID.
```

> # Amplify Access Logs Shipper - Lambda

Reviewer 2: Please have @nico-shishkin review this README, and add to the docs as well.

Author: @nico-shishkin  can you go over of Readme.

This is an AWS Lambda function that collects Amplify access logs and sends them to Logz.io in bulk over HTTP.

-   Option 1: [Manual Lambda configuration](#manual-lambda-configuration)

> <!-- - Option 2: [Automated CloudFormation deployment](#automated-cloudformation-deployment) -->

Reviewer 2: If at the moment there is only one option, we don't need this.

<div id="manual-lambda-configuration">

> ## Manual configuration with a Lambda function

Reviewer 2: If at the moment there is only one option, we don't need this.
